react/src/lib/ComboBox/index.js

```diff
@@ -276,7 +276,7 @@ class ComboBox extends React.Component {
           'md-combo-box' +
           `${(className && ` ${className}`) || ''}`
         }
-        role='combobox'
+        role='search'
```

Comment:
Is combobox not a valid role?

Reply:
Screen reader has issue with combobox role. It default change input to browse mode in which input unable to receive text and all type text send to screen reader.